imagebitmap-renderingcontext/context-creation-offscreen.html

```diff
@@ -5,13 +5,14 @@
 <script src="/resources/testharnessreport.js"></script>
 <link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#the-imagebitmap-rendering-context">
```

Comment:
```suggestion
 <link rel="help" href="https://html.spec.whatwg.org/multipage/canvas.html#the-imagebitmap-rendering-context">
```

The old URL redirects here, so it's not a big issue, but it was a bit confusing when I was reviewing. Made me think that the spec was spread across multiple files, since the URLs in the tests didn't match the spec I was looking at.

Same issue on the other file, but the `<link>` in that case isn't showing up in the PR changes list so I can't make it a "suggestion".